homeassistant/components/wallbox/number.py

```diff
@@ -26,15 +28,70 @@
 from .entity import WallboxEntity
 
 
+def max_charging_current_value(coordinator: WallboxCoordinator) -> float:
+    """Return the maximum available value for charging current."""
+    return cast(float, coordinator.data[CHARGER_MAX_AVAILABLE_POWER_KEY])
+
+
+def min_charging_current_value(coordinator: WallboxCoordinator) -> float:
+    """Return the minimum available value for charging current."""
+    return cast(
+        float,
+        (
+            coordinator.data[CHARGER_MAX_AVAILABLE_POWER_KEY] * -1
+            if coordinator.data[CHARGER_DATA_KEY][CHARGER_PART_NUMBER_KEY][0:2]
+            in BIDIRECTIONAL_MODEL_PREFIXES
+            else 0
+        ),
+    )
```

Comment:
Can we maybe get the if statement out of the cast?

Reply:
sure